heka/_common.sls

```diff
@@ -8,6 +8,15 @@ heka_packages:
   file.recurse:
   - source: salt://heka/files/lua
 
+/usr/share/lma_collector/common/extra_fields.lua:
```

Comment:
You shouldn't touch system package managed locations from Salt. Recommended location for these things is using `/usr/local` prefix, in this case `/usr/local/share/lma_collector/...`